util/network-devp2p/src/session.rs

```diff
@@ -281,15 +280,15 @@ impl Session {
 		};
 		let mut rlp = RlpStream::new();
 		rlp.append(&(u32::from(pid)));
-		let mut compressed = Vec::new();
+		let compressed;
```

Comment:
move the `compressed variable` inside the `if branch`?

Reply:
It has to outlive `payload`. Probably possible to refactor this, but decided to go with the least invasive route.

Reply:
Ok, make sense